test/python/compiler/test_assembler.py

```diff
@@ -548,6 +548,16 @@ def test_pulse_gates_common_cals(self):
         self.assertEqual(len(qobj.experiments[0].config.calibrations.gates), 1)
         self.assertFalse(hasattr(qobj.experiments[1].config, 'calibrations'))
 
+    def test_pulse_gates_delay_only(self):
+        """Test that a single delay gate is translated to an instruction."""
+        circ = QuantumCircuit(2)
+        circ.append(Gate('test', 1, []), [0])
+        circ.add_calibration('test', [0], pulse.Delay(160, DriveChannel(0)))
+        qobj = assemble(circ, FakeOpenPulse2Q())
+        self.assertEqual(len(qobj.config.calibrations.gates[0].instructions), 1)
+        self.assertEqual(qobj.config.calibrations.gates[0].instructions[0].to_dict(),
```

Comment:
Maybe you can test w/ a `shift` to verify that works? Ie put an instruction before and make sure the time gets shifted.